python/sdk/merlin/observability/inference.py

```diff
@@ -250,3 +269,11 @@ class InferenceSchema:
     @property
     def feature_columns(self) -> List[str]:
         return list(self.feature_types.keys())
+
+    @property
+    def prediction_columns(self) -> List[str]:
+        return list(self.model_prediction_output.prediction_types().keys())
+
+    @property
+    def ground_truth_columns(self) -> List[str]:
```

Comment:
who call this property?

Reply:
Not in this PR, but will be used by flyte workflow that publish ground truth eventually.